File: solutions/beecrowd/1043/1043.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[a b c] (->> (str/split line #" ")
                         (map #(Double/parseDouble %)))]
        (if (and (< a (+ b c))
                 (< b (+ a c))
                 (< c (+ a b)))
          (printf "Perimetro = %.1f%n" (+ a b c))
          (printf "Area = %.1f%n" (/ (* c (+ a b)) 2))))
      (recur (read-line)))))

(main)

File: solutions/beecrowd/1043/1043.py
import sys

for line in sys.stdin:
    a, b, c = map(float, line.split())

    if a < b + c and b < a + c and c < a + b:
        print(f'Perimetro = {a + b + c:.1f}')
    else:
        print(f'Area = {c * (a + b) / 2:.1f}')
